<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureSessionsWidget</class>
 <widget class="QWidget" name="ConfigureSessionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Administraton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableSessions">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Enable session management</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delete sessions data</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="delOlderOneMonth">
       <property name="text">
        <string>Delete data older than one month</string>
       </property>
       <attribute name="buttonGroup">
        <string>dateGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="delOlderSixMonth">
       <property name="text">
        <string>Delete data older than six months</string>
       </property>
       <attribute name="buttonGroup">
        <string>dateGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="deleteSpecifiedDate">
       <property name="text">
        <string>Delete data older than the specified date</string>
       </property>
       <attribute name="buttonGroup">
        <string>dateGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="dateLimit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="deleteFilesInfo">
       <property name="text">
        <string>Delete information on files (the files will not be touched)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteData">
       <property name="text">
        <string>Delete sessions data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="deleteAllData">
       <property name="text">
        <string>Delete all sessions data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>deleteAllData</zorder>
  <zorder>enableSessions</zorder>
  <zorder>line</zorder>
  <zorder>label_2</zorder>
  <zorder>horizontalSpacer_3</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="dateGroup"/>
 </buttongroups>
</ui>
